src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -625,6 +625,136 @@ impl ToQemuParams for DeviceNvdimm {
     }
 }
 
+#[derive(Debug)]
+struct BlockDriver {
```

Comment:
I'm not sure what the status of `-drive` is in qemu, would it be better use `-blockdev` and `-device` instead? (@gkurz )

If we stick with `-drive`, please rename this class to `Drive` (the idea of the naming convention used in this file is for the class names to be as close as possible to the cmdline param they generate).

Reply:
> I'm not sure what the status of `-drive` is in qemu, would it be better use `-blockdev` and `-device` instead? (@gkurz )
> 

`-drive` is legacy. Even if it is still supported, `-blockdev` and `-device` is the preferred way.

> If we stick with `-drive`, please rename this class to `Drive` (the idea of the naming convention used in this file is for the class names to be as close as possible to the cmdline param they generate).

Problem with `-drive` is that it conflates both the guest visible part (the `-device`) and the host backend part (the `-blockdev`), i.e. QEMU creates the device and backend internally, but it doesn't allow the same fine grained configuration you get with `-device` and `-blockdev`.

Reply:
I switched to the `-blockdev` and `-device` combination. Thanks!